openfe/setup/network.py

```diff
@@ -41,14 +41,14 @@ def graph(self) -> nx.Graph:
         return self._graph
 
     @property
-    def edges(self) -> List[AtomMapping]:
-        """List of edges"""
-        return self._edges
+    def edges(self) -> Iterable[AtomMapping]:
```

Comment:
My understanding is that generators are preferred to avoid *creating* a container that takes memory. The memory is already allocated here -- any reason not to just return the `frozenset` instead of a consumable iterator? (which, because it is consumable, is a potential source of bugs/user error) Typing could be `typing.FrozenSet`? (I know that in Python 3.9, we can even use `frozenset[AtomMapping]`, but I'm not sure if that works in 3.8).

Previous type declaration was wrong, and should have been a frozenset, not list.